code/VersionedGridFieldOrderableRows.php

```diff
@@ -112,26 +128,52 @@ public function augmentColumns($grid, &$cols)
         }
     }
 
+    /**
+     * @param $grid
+     * @return array
+     */
     public function getColumnsHandled($grid)
     {
         return array('Reorder');
     }
 
+    /**
+     * @param $grid
+     * @param $record
+     * @param $col
+     * @return mixed
+     */
     public function getColumnContent($grid, $record, $col)
     {
         return ViewableData::create()->renderWith('VersionedGridFieldOrderableRowsDragHandle');
     }
 
+    /**
+     * @param $grid
+     * @param $record
+     * @param $col
+     * @return array
+     */
     public function getColumnAttributes($grid, $record, $col)
     {
         return array('class' => 'col-reorder');
     }
 
+    /**
+     * @param $grid

```

Comment:
I don't know if these or the above are adding any value if they don't have the types